components/script/dom/htmlimageelement.rs

```diff
@@ -1420,6 +1420,21 @@ pub fn parse_a_sizes_attribute(value: DOMString) -> SourceSizeList {
     SourceSizeList::parse(&context, &mut parser)
 }
 
+pub(self) fn get_correct_referrerpolicy_from_raw_token(token: &DOMString) -> DOMString {
```

Comment:
What's the purpose of `pub(self)`?

Reply:
I didn't compile without this line before. Thank you for helping me with this!